app/src/main/java/com/infomaniak/mail/data/cache/mailboxContent/DraftController.kt

```diff
@@ -120,7 +121,7 @@ class DraftController @Inject constructor(
         }
 
         fun updateDraft(localUuid: String, realm: MutableRealm, onUpdate: (Draft) -> Unit) {
```

Comment:
Shouldn't we name it updateDraftBlocking to pass the knowledge that it's using blocking code ?